include/oneapi/mkl/detail/backends.hpp

```diff
@@ -36,17 +36,19 @@ enum class backend {
     netlib,
     rocblas,
     rocrand,
+    syclblas,
```

Comment:
Can we build new backend with the name of what device arch it is targeted for? E.g., `syclblas_intelgpu`, `syclblas_nvidiagpu`? With the current implementation it's not possible to have syclblas backends built for different targets at the same time.

Reply:
Not really, I'm just trying to clarify the long-term support. If the dispatching between different HW will happen in the backend itself it's fine to me to have only `syclblas` as the backend library name. But in this case I guess all dispatching logic implemented on oneMKL Interfaces side will be duplicated inside `syclblas` backend, so we probably need to make sure we will remove this duplication in the future.